src/index.html

```diff
@@ -4,10 +4,55 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="device-width initial-scale=1">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <section class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__logo"
+        >
+
+        <h2 class="card__description">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+          </div>
+
+          <p class="card__review-text">Reviews: 5</p>
```

Comment:
Move this line out of `stars` block (just place it below the next closing `div`)

Reply:
I need help. I`m lost in the task.